src/fmgc/src/efis/EfisCommon.ts

```diff
@@ -1,13 +1,18 @@
-import { Mode, RangeSetting } from '@shared/NavigationDisplay';
+// Copyright (c) 2021-2022 FlyByWire Simulations
+// Copyright (c) 2021-2022 Synaptic Simulations
```

Comment:
Why are synaptic claiming copyright on this?

Reply:
Fixed